docs/howto/team-open-science-mindset.md

```diff
@@ -21,46 +21,77 @@ We want our community ontology editors and developers to be _excited_ to be part
+    - Can improve the materials by fixing it (please feel empowered to fix anything, including typos and content).
+
+<img width="1080" alt="image" src="https://user-images.githubusercontent.com/6722114/190459617-ba48bafd-e4ef-44e2-ac6e-d70de1cd8147.png">
 
 1. **Be an open scientist**
 
-- Contribute things openly so other people can benefit from it
-- ie don't put issues on Slack where the public can't see it.
-- Create public tickets
+    - Contribute things openly so other people can benefit from it
+    - Don't put issues on Slack where the public can't see it
+    - Create public tickets
+    - Create tickets on external ontology trackers when you spot issues in other ontologies
+
+Example: While curating Mondo, Nicole identified issues relevant to Orphanet and created this [issue](https://github.com/Orphanet/ORDO/issues/31).
+
+<img width="1265" alt="image" src="https://user-images.githubusercontent.com/6722114/190462261-3a46a3aa-1cec-4f1e-a77d-cf5fef16b34e.png">
+
+
+Example: There is overlap between Mondo and Human Phenotype Ontology and the Mondo and HPO curators tag each other on relevant tickets.
+
+<img width="913" alt="image" src="https://user-images.githubusercontent.com/6722114/190462558-18b7de10-0ebe-4f27-aaf3-2c6655d3fb80.png">
+
+Example: In Mondo, if new classifications are made, Mondo curators report this back to the source ontology to see if they would like to follow our classification.
+
+<img width="1263" alt="image" src="https://user-images.githubusercontent.com/6722114/190462954-813104d2-23c7-4fd2-99e6-6b3f2f00fde5.png">
+
 
 1. **Find a buddy**
 
-- Pair up with someone who will review your PRs and you will review their PRs
+    - Pair up with someone who will review your PRs and you will review their PRs
 
 1. **Be clear and inclusive**
 
-- Take the time to write proper, detailed responses, avoid being vague or terse
-- Sloppy, quick responses can cause more work than not responding at all
-- Bullet lists, checklists for action items
-- Link to external issues
-- Hide unnecessary details and with details tag: <details><summary>[click arrow to expand]</summary>. See [example here](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1978#issuecomment-1183176682)
-- Undercut any aggressive comments
+    - Take the time to write proper, detailed responses, avoid being vague or terse
+    - Sloppy, quick responses can cause more work than not responding at all
+    - Bullet lists, checklists for action items
+    - Hide unnecessary details and with details tag: <details><summary>[click arrow to expand]</summary>. See [example here](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1978#issuecomment-1183176682)
+    - Undercut any aggressive comments
+    - Link to external issues
+
+Example:
+
+<img width="643" alt="image" src="https://user-images.githubusercontent.com/6722114/190461543-4be5e98e-0436-4d1a-ad91-60e371a71eff.png">
 
 1. **Reduce work for others as others as much as possible**
```

Comment:
Remove "as others".
Although, I would not take this instruction at face value considering that if I always reduced work for others, I would never finish my own work :)